File: UI/afk_android/lib/models/todo4924.dart

  import 'package:json_annotation/json_annotation.dart';
part 'todo4924.g.dart';

@JsonSerializable()
class ToDo4924{
  int? todo4924id ;

String? nazivaktivnosti;

String? opisaktivnosti;
int? korisnikid;
DateTime? datumzavrsenja;
String? statemachine;


  ToDo4924(this.todo4924id, this.nazivaktivnosti, this.opisaktivnosti, this.korisnikid, this.datumzavrsenja, this.statemachine);
  // factory Korisnici.fromJson(Map<String,dynamic>json)=>_$KorisniciFromJson(json);

  factory ToDo4924.fromJson(Map<String,dynamic>json)=>_$ToDo4924FromJson(json);
  Map<String,dynamic>toJson()=>_$ToDo4924ToJson(this);
}

File: FudbalskiKlub.Model/ToDo4924.cs
using System;
using System.Collections.Generic;

namespace FudbalskiKlub.Services.Model
{
    public partial class ToDo4924
    {
        public int todo4924id { get; set; }

        public string? nazivaktivnosti { get; set; }

        public string? opisaktivnosti { get; set; }

        public DateTime datumzavrsenja { get; set; }

        public int korisnikid { get; set; }
        public string? statemachine { get; set; }

    }

}

